nntrainer/graph/network_graph.cpp

```diff
@@ -344,10 +344,21 @@ void NetworkGraph::applyGradients(
   }
 }
 
-sharedConstTensors
-NetworkGraph::forwarding(bool training,
-                         std::function<bool(void *userdata)> stop_cb) {
-  for (auto iter = cbegin(); iter != cend() && !stop_cb(nullptr); iter++) {
+sharedConstTensors NetworkGraph::forwarding(
+  ml::train::RunStats &stat, bool training,
```

Comment:
We do not need this stat here. May be move it to end of function parameters and set default equal nullptr.